src/com/company/Team.java

```diff
@@ -0,0 +1,80 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Team {
+    private int teamId;
+    private String name;
+    private int noOfPlayers;
+    private int score=0;
+    private int notOut;
```

Comment:
instead of maintaining notOut, maintain wickets

Reply:
Done